apps/ideaspace/src/components/modules/WorkshoppingPage/IdeaOverview/IdeaCard/IdeaCard.js

```diff
@@ -12,44 +13,35 @@ import {
   BottomView,
   Button,
   StyledText,
-} from './StyledIdeaCard'
+} from './StyledIdeaCard';
 
 export const IdeaCard = ({ ideaImage, ideaName, ideaTagLine }) => {
   return (
     <StyledCard>
       <TopView>
         <LeftView>
-          {
-            ideaImage ? <img src={ideaImage}/> : 
-            <StyledDiv>
-            </StyledDiv>
-          }
-          
+          {ideaImage ? <img src={ideaImage} /> : <StyledDiv></StyledDiv>}
         </LeftView>
         <RightView>
-          <IdeaName>
-            {ideaName}
-          </IdeaName>
-          <IdeaTagLine>
-            {ideaTagLine}
-          </IdeaTagLine>
+          <IdeaName>{ideaName}</IdeaName>
+          <IdeaTagLine>{ideaTagLine}</IdeaTagLine>
         </RightView>
       </TopView>
 
-      {/* <BottomView>
+      <BottomView>
         <Button>
           <BookmarkBorderOutlinedIcon />
-          <StyledText>Save</StyledText>
+          <StyledText>FOLLOW</StyledText>
         </Button>
         <Button>
-          <ShareOutlinedIcon />
-          <StyledText>Share</StyledText>
+          <StarBorderOutlinedIcon />
+          <StyledText>UPVOTE</StyledText>
         </Button>
         <Button>
-          <FavoriteBorderOutlinedIcon />
-          <StyledText>Like</StyledText>
+          <ShareOutlinedIcon />
+          <StyledText>SHARE</StyledText>
```

Comment:
Can we keep the SHARE & FOLLOW buttons hidden until that functionality is built out? Adding them here is outside the scope of this ticket.